source/include/dqm4hep/Application.h

```diff
@@ -512,6 +514,25 @@ namespace dqm4hep {
     //-------------------------------------------------------------------------------------------------
     //-------------------------------------------------------------------------------------------------
     
+    template <typename T>
+    void Application::sendStat(const std::string &entryName, const T &stats){
```

Comment:
You need `inline` here or you might linking problems ...

Reply:
Sure!